README.md

```diff
@@ -1 +1,17 @@
-# Hello-World
\ No newline at end of file
+# Hello-World
+This is my first project with microverse, where i needed to create 3 files Html, css, and test.md the last file needed to be ignored, I have completed the project as per requirments. 
+## built with 
+HTML
+CSS
+
+## Getting started: 
+all you need to do is to just open the HTML file, copy it to your local machine and test it out, this is a simple project. 
+
+## Authors 
+Github: @shakerAbuDrais
+
+## 🤝 Contributing
+## Show your support
+Give a ⭐️ if you like this project!
+
+# 📝 License
```

Comment:
You have done well updating your readme 😄, but I think you should improve on your readme by following the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. Please ensure your readme looks like the template provided 👍🏼 .